provisioning/ansible/roles/external_mail_handler/tasks/main.yml

```diff
@@ -4,6 +4,18 @@
 # Additionally, creates a logrotate config file for logs
 #
 
+- name: Upgrade binutils
```

Comment:
A principal idea of SOCBED is to pin as many package versions as possible to (1) ensure reproducibility and (2) avoid unforeseen problems in the future. I'm actually not sure how many packages and other software are currently affected by this, we should discuss whether is makes sense to pin for example this package version here.